spearhead/assets/sass/style.scss

```diff
@@ -14,7 +14,7 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 */
 
 @import "../../../seedlet/assets/sass/abstracts/mixins";
-@import "../../../seedlet/assets/sass/base/accesibility";
+@import "../../../seedlet/assets/sass/base/accessibility";
```

Comment:
I think we should import these where they are actually needed

Reply:
That would be here, sorry for breaking things!

https://github.com/Automattic/themes/blob/49eead899692a426a2ad97caf0665f04c7f5bb8b/spearhead/assets/sass/_search.scss#L1

Reply:
No worries, thanks!